docs/for_linux_developers.md

```diff
@@ -25,7 +25,7 @@ If you want to install only main dependencies, you can use:
 poetry  install --only main,test
 ```
 
-### Building and running the app
+## Building and running the app
 
 Application consists of multiple components. The simplest way to run the app is to use docker-compose. Following instruction will guide you through the process of set up and running the application.
```

Comment:
Maybe you could explain the connections between components using simple mermaid flowchart ?
https://mermaid.js.org/syntax/flowchart.html